README.md

```diff
@@ -1,14 +1,14 @@
 <b>[English](README.md)</b> (recommended) | [中国语文科](README.zh.md) | [Русский](README.ru.md)
 
 
-# JavaScript captcha solver module for 2Captcha API
```

Comment:
This package does not specifically concern JS only anymore, which is why I changed this.